aip/client-libraries/4223.md

```diff
@@ -0,0 +1,39 @@
+---
+id: 4223
+state: draft # TODO(noahdietz): Update pre-merge
+created: 2022-05-11 # TODO(noahdietz): Update pre-merge
+---
+
+# Client-side paylod validation
+
+Services document the behavior of specific request fields using the
```

Comment:
This paragraph, on the face of it, does not tie in to the title of its section, though as we keep reading we realize it provides background. So I would suggest we move this information down to the "Motivation" section, and replace it here with something like:

"This AIP describes the role of client libraries in validating request contents before they are sent to the server."

Reply:
In other AIPs the section just before the Guidance provides an introduction, and in practice, some context, for the guidance (https://aip.dev/8#document-structure). I think I'd prefer to keep this paragraph here ,but improve it to be more relevant. Is that reasonable?

Reply:
Sure, if that's the convention. Let's tie it in to **client-side** validation.

Reply:
Sure thing, I've updated it, please take a look!